docs/architecture/anti-entropy/hinted-handoff.rst

```diff
@@ -43,4 +43,63 @@ Hinted handoff is enabled and managed by these settings in :code:`scylla.yaml`:
 
 Storing of the hint can also fail. Enabling hinted handoff therefore does not eliminate the need for repair; a user must recurrently :doc:`run a full repair </operating-scylla/procedures/maintenance/repair/>` to ensure data consistency across the cluster nodes.
 
+Sync point HTTP API
+^^^^^^^^^^^^^^^^^^^
```

Comment:
We have API reference we can point to https://opensource.docs.scylladb.com/stable/reference/api-reference.html#hinted-handoff

(can be in a followup PR)

Reply:
Just to make sure I understand – you want to have the header be a link to that article, correct?

Reply:
> Just to make sure I understand – you want to have the header be a link to that article, correct? 

Yes, it would be useful to let users know about the API reference.

Reply:
scylladb/scylladb#20065